packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -80,7 +81,7 @@ export const ContextualMenu = compose<ContextualMenuType>({
   },
   settings: settings,
   slots: {
-    root: Callout,
+    root: View,
```

Comment:
Is this change intentional?

Reply:
No.... good catch

Reply:
This is my super-not-so-secret trick for testing stuff without a native module, which should definitely not be committed.